.gitignore

```diff
@@ -37,4 +37,4 @@
 /node_modules
 /yarn-error.log
 yarn-debug.log*
-.yarn-integrity
+.yarn-integrity
```

Comment:
it's a good practice to leave an empty line at the end of the file
https://stackoverflow.com/a/5813359/4983910